first-time.sql

```diff
@@ -0,0 +1,298 @@
+  `language` varchar(45) NOT NULL,
+  `data_cache` blob NOT NULL,
+  `user_link_id` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`wiki`,`user_id`),
+  KEY `USERNAME` (`user_name`),
+  KEY `LASTLOGIN` (`last_login`),
+  KEY `LASTACTION` (`last_action`),
+  KEY `BLOCKED` (`blocked`),
+  KEY `LINKID` (`user_link_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `externallinks_userflags`
+--
+
+DROP TABLE IF EXISTS `externallinks_userflags`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `externallinks_userflags` (
+  `user_id` int(10) unsigned NOT NULL,
+  `wiki` varchar(45) NOT NULL,
+  `user_flag` varchar(255) NOT NULL,
+  KEY `USERID` (`wiki`,`user_id`),
+  KEY `FLAGS` (`user_flag`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `externallinks_userflags`
+--
+
+LOCK TABLES `externallinks_userflags` WRITE;
+/*!40000 ALTER TABLE `externallinks_userflags` DISABLE KEYS */;
+/*!40000 ALTER TABLE `externallinks_userflags` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `externallinks_userlog`
+--
+
+DROP TABLE IF EXISTS `externallinks_userlog`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `externallinks_userlog` (
+  `log_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `wiki` varchar(45) NOT NULL,
+  `locale` varchar(45) NOT NULL,
+  `log_type` varchar(45) NOT NULL,
+  `log_action` varchar(45) NOT NULL,
+  `log_object` bigint(12) NOT NULL,
+  `log_object_text` blob DEFAULT NULL,
+  `log_from` blob DEFAULT NULL,
+  `log_to` blob DEFAULT NULL,
+  `log_user` int(11) NOT NULL,
+  `log_timestamp` timestamp NOT NULL DEFAULT current_timestamp(),
+  `log_reason` varchar(255) NOT NULL DEFAULT '',
+  PRIMARY KEY (`log_id`),
+  KEY `WIKI` (`wiki`),
+  KEY `LOCALE` (`locale`),
+  KEY `LOGTYPE` (`log_type`),
+  KEY `LOGACTION` (`log_action`),
+  KEY `LOGOBJECT` (`log_object`),
+  KEY `LOGUSER` (`log_user`),
+  KEY `LOGTIMESTAMP` (`log_timestamp`),
+  KEY `LOGSPECIFIC` (`log_type`,`log_action`,`log_object`,`log_user`)
+) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `externallinks_userlog`
+--
+
+LOCK TABLES `externallinks_userlog` WRITE;
+/*!40000 ALTER TABLE `externallinks_userlog` DISABLE KEYS */;
+INSERT INTO `externallinks_userlog` VALUES (1,'testwiki','testwiki','tos','accept',0,'',NULL,NULL,1,'2021-06-16 23:40:57',''),(2,'testwiki','testwiki','wikiconfig','change',0,'',NULL,NULL,1,'2021-06-17 00:05:39',''),(3,'testwiki','testwiki','wikiconfig','change',0,'',NULL,NULL,1,'2021-06-17 00:06:13',''),(4,'global','testwiki','usergroups','define',0,'',NULL,NULL,1,'2021-06-17 00:07:43','');
```

Comment:
I would dump this line.  Let's leave the activity log empty for new dev users.

Reply:
Fixed